.github/scripts/version_manager.sh

```
@@ -347,9 +452,13 @@ update_project_file() {
 
     case "$PROJECT_TYPE" in
         "spring")
-            # build.gradle 업데이트
-            sed -i.bak "s/version = '.*'/version = '$new_version'/" "$VERSION_FILE"
-            rm -f "${VERSION_FILE}.bak"
+            # 모든 모듈의 build.gradle 업데이트 (루트 포함)
+            for f in $(find . -maxdepth 2 -name build.gradle -type f); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)